File: Utopia-Commands/scripts/hello_world.py
#Works with Jython2.7a2
from api.commands import CommandFactory, CommandResponse


def handles():
    return CommandFactory.newSimpleCommand("jython")


def handleCommand(context, params, filters, delayedEventPoster):
    return CommandResponse.resultResponse("helloWorld", "Hello World from Jython")

File: Utopia-API/scripts/hello_world.js
importPackage(Packages.api.commands);

function handles() {
    return CommandBuilder.forCommand("js").build();
}

function handleCommand(context, params, filters, delayedEventPoster) {
    var name = "world";
    if (params.containsKey("name")) name = params.getParameter("name");
    return CommandResponse.resultResponse("helloWorld", "Hello " + name + " from javascript");
}

function getParsers() {
    var parsers = java.lang.reflect.Array.newInstance(CommandParser, 2);
    parsers[0] = CommandParser(ParamParsingSpecification("name", "[^ ]+"));
    parsers[1] = CommandParser.getEmptyParser();
    return parsers;
}